joint_trajectory_action:
  joints:
    - joint_1
    - joint_2
    - joint_3

  constraints:
    goal_time: 0.3
    joint_1:
      goal: 0.04
    joint_2:
      goal: 0.04
    joint_3:
      goal: 0.04